shared/gh/js/views/gh.visibility.js

```diff
@@ -26,18 +34,39 @@ define(['gh.core'], function(gh) {
     };
 
     /**
+     * Dismiss the 'visibility' modal dialog
+     *
+     * @private
+     */
+    var dismissVisibilityModal = function() {
+        $('#gh-visibility-modal').modal('hide');
+    };
+
+    /**
      * Render and show the 'visibility' modal dialog
      *
      * @private
      */
     var showVisibilityModal = function() {
-        // Render the modal
-        gh.utils.renderTemplate($('#gh-visibility-modal-template'), {'data': {
-            // TODO: make this dynamic once the functionality is available in the back-end
-            'visibility': 'draft'
-        }}, $('#gh-visibility-modal-container'));
-        // Show the modal
-        $('#gh-visibility-modal').modal();
+        // Cache the trigger
+        var $trigger = $(this);
+        // Retrieve the organisational unit ID
+        orgUnitId = parseInt($trigger.data('id'), 10);
```

Comment:
This hasn't been updated yet per previous comment. Instead of caching the ID we could retrieve it from the hash.